docs/csdk-guide.md

```diff
@@ -29,6 +29,32 @@ code and without CarbonSession.
 
 In the carbon jars package, there exist a carbondata-sdk.jar, 
 including SDK reader for C++ SDK.
+
+##Compile/Build CSDK
+CSDK supports cmake based compilation and has dependency list in CMakeLists.txt.
+<br> Prerequisites
+<br>GCC >=4.8.5
+<br>Cmake >3.13
+<br>Make >=4.1
+
+Steps 
+1. Go to CSDK folder(/opt/.../CSDK/) 
+2. Create build folder . (/opt/.../CSDK/build) 
+3. Run Command from build folder `cmake ../`
+4. `make`
+
+Test Cases are written in  [main.cpp](https://github.com/apache/carbondata/blob/master/store/CSDK/test/main.cpp) with GoogleTest C++ Framework.
+if GoogleTest LIBRARY is not added then compilation of example code will fail. Please follow below steps to solve the same
```

Comment:
please change if to If, LIBRARY to library. Please take care the upper/lower case.

Reply:
Done for required words.